terraform/gcp/cluster.tf

```diff
@@ -9,6 +9,32 @@ resource "google_container_cluster" "cluster" {
   initial_node_count       = 1
   remove_default_node_pool = true
 
+  // For private clusters, pass the name of the network and subnetwork created
+  // by the VPC
+  network    = var.enable_private_cluster ? google_compute_network.vpc_network[0].self_link : null
```

Comment:
Since we use `.name` rather than `.self_link` in other places, let's use `.name` consistently?

Reply:
Ah, I did not realise they were equivalent :)

Reply:
I don't know if they're always equivalent, but `network` definitely accepts either https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#network